Sources/Socket.swift

```diff
@@ -672,31 +678,31 @@ extension Socket {
 }
 
 public enum SecurityMechanism {
-    case Null
-    case Plain
-    case CURVE
+    case null
+    case plain
+    case curve
 }
 
 extension SecurityMechanism {
     init?(rawValue: Int32) {
         switch rawValue {
-        case ZMQ_NULL: self = Null
-        case ZMQ_PLAIN: self = Plain
-        case ZMQ_CURVE: self = CURVE
+        case ZMQ_NULL: self = .null
+        case ZMQ_PLAIN: self = .plain
+        case ZMQ_CURVE: self = .curve
         default: return nil
         }
     }
 }
 
 extension Socket {
     public func sendString(_ string: String, mode: SendMode = []) throws -> Bool {
```

Comment:
Just `send` is good here - the type information is enough (following the API design guidelines). Same goes for `receiveString`